layouts/index.html

```diff
@@ -225,7 +225,7 @@ <h3>Source code</h3>
                     </div>
                     <div class="row">
                         <div class="col-md-12 top-space">
-                            <p class="center-text">(*) Looking for older or beta releases? Have a look in our <a href="/releases/">release archive</a></p>
+                            <p class="center-text">(*) Looking for older or beta releases? Have a look in our <a class="link-text" href="/releases/">release archive</a></p>
```

Comment:
I'm not sure I understand why we need a `link-text` class to match `<a>` elements. Which links would we *not* mark up with the new `link-text` class?

Reply:
Fair point! Will check